src/components/gallery/Picture.jsx

```diff
@@ -1,8 +1,22 @@
-const Picture = () => {
+"use client";
+import React, { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+const Picture = ({ text, link, image }) => {
+  const [isHovered, setIsHovered] = useState(false);
+
   return (
-    <div className="">
-      <p>Picture</p>
-    </div>
+    <Link href={link} target="_blank">
+      <div className="relative flex justify-center items-center">
+        {isHovered && <p className="absolute">{text}</p>}
```

Comment:
you should still style the component. you don't have to code in the outline or the font, but the component should generally look like the example from the issue